img/j/joinmastodon.org.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 216.4 232"><path fill="#2b90d9" d="M211.8 139c-3.2 16.5-28.5 34.4-57.6 37.8a278 278 0 0 1-46 2.8c-26-1.2-46.5-6.2-46.5-6.2 0 2.5.1 5 .4 7.2 3.4 25.7 25.5 27.2 46.4 28 21.2.6 40-5.3 40-5.3l.8 19s-14.7 8-41 9.5a171 171 0 0 1-53.6-6C9.2 213.9 1.4 165.4.2 116.2-.2 101.5.1 87.7.1 76c0-50.3 33-65 33-65C49.6 3.5 78.1.2 107.8 0h.7c29.7.2 58.2 3.5 74.8 11 0 0 33 14.8 33 65.2 0 0 .4 37.1-4.6 62.9"/><path fill="#fff" d="M177.5 80v61h-24.1V82c0-12.5-5.3-18.8-15.8-18.8-11.6 0-17.4 7.5-17.4 22.3v32.4h-24V85.4c0-14.8-5.8-22.3-17.4-22.3C68.3 63 63 69.4 63 81.9V141H39V80c0-12.4 3.2-22.3 9.5-29.6 6.6-7.3 15.2-11 25.9-11a31 31 0 0 1 27.9 14.2l6 10 6-10a31.1 31.1 0 0 1 28-14.3c10.6 0 19.2 3.8 25.8 11.1a43.7 43.7 0 0 1 9.5 29.7"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 75 79"><path fill="url(#a)" d="M73.84 17.49C72.7 9 65.3 2.31 56.53 1.02 55.05.8 49.44 0 36.46 0h-.1C23.37 0 20.6.8 19.11 1.02 10.58 2.28 2.79 8.29.9 16.88 0 21.1-.1 25.79.06 30.1c.24 6.17.29 12.32.85 18.47a86.88 86.88 0 0 0 2.02 12.11c1.8 7.36 9.06 13.49 16.18 15.99a43.41 43.41 0 0 0 23.68 1.25c.87-.2 1.72-.44 2.56-.7 1.91-.61 4.15-1.29 5.8-2.48a.19.19 0 0 0 .07-.15v-5.95a.18.18 0 0 0-.21-.17 65.64 65.64 0 0 1-15.37 1.8c-8.9 0-11.3-4.23-11.99-5.99a18.55 18.55 0 0 1-1.02-4.8.17.17 0 0 1 .12-.1h.08a64.38 64.38 0 0 0 15.11 1.8c1.23 0 2.45 0 3.67-.03 5.12-.14 10.52-.4 15.56-1.39.13-.02.25-.04.36-.08 7.95-1.52 15.5-6.31 16.28-18.44.03-.48.1-5 .1-5.5 0-1.68.54-11.95-.08-18.25Z"/><path fill="#fff" d="M61.25 27.03V48.1h-8.36V27.65c0-4.31-1.8-6.5-5.45-6.5-4.01 0-6.02 2.6-6.02 7.73v11.2h-8.3v-11.2c0-5.14-2.02-7.74-6.04-7.74-3.63 0-5.44 2.2-5.44 6.5v20.47h-8.36V27.03c0-4.31 1.1-7.74 3.3-10.27 2.27-2.53 5.25-3.83 8.95-3.83 4.27 0 7.5 1.64 9.66 4.93l2.08 3.49 2.08-3.5c2.16-3.28 5.39-4.92 9.66-4.92 3.7 0 6.67 1.3 8.95 3.83 2.2 2.53 3.3 5.96 3.29 10.27Z"/><defs><linearGradient id="a" x1="37.07" x2="37.07" y1="0" y2="79" gradientUnits="userSpaceOnUse"><stop stop-color="#6364FF"/><stop offset="1" stop-color="#563ACC"/></linearGradient></defs></svg>
```

Comment:
Applying the percentages of X and Y coordinates (of the gradient) to the original dimensions should work alright:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 216.4 232"><path fill="url(#a)" d="M211.8 139c-3.2 16.5-28.5 34.4-57.6 37.8a278 278 0 0 1-46 2.8c-26-1.2-46.5-6.2-46.5-6.2 0 2.5.1 5 .4 7.2 3.4 25.7 25.5 27.2 46.4 28 21.2.6 40-5.3 40-5.3l.8 19s-14.7 8-41 9.5a171 171 0 0 1-53.6-6C9.2 213.9 1.4 165.4.2 116.2-.2 101.5.1 87.7.1 76c0-50.3 33-65 33-65C49.6 3.5 78.1.2 107.8 0h.7c29.7.2 58.2 3.5 74.8 11 0 0 33 14.8 33 65.2 0 0 .4 37.1-4.6 62.9"/><path fill="#fff" d="M177.5 80v61h-24.1V82c0-12.5-5.3-18.8-15.8-18.8-11.6 0-17.4 7.5-17.4 22.3v32.4h-24V85.4c0-14.8-5.8-22.3-17.4-22.3C68.3 63 63 69.4 63 81.9V141H39V80c0-12.4 3.2-22.3 9.5-29.6 6.6-7.3 15.2-11 25.9-11a31 31 0 0 1 27.9 14.2l6 10 6-10a31.1 31.1 0 0 1 28-14.3c10.6 0 19.2 3.8 25.8 11.1a43.7 43.7 0 0 1 9.5 29.7"/><defs><linearGradient id="a" x1="107" x2="107" y1="0" y2="232" gradientUnits="userSpaceOnUse"><stop stop-color="#6364FF"/><stop offset="1" stop-color="#563ACC"/></linearGradient></defs></svg>
```